conf/ds-discover-behaviour.yaml

```diff
@@ -7,7 +7,7 @@
 #
 # This config is controlled by the individual developer and is not part of the code repository.
 #
-
+url: http://localhost:9074/ds-discover/
```

Comment:
This should be under some sort of grouping. Maybe
```
openapi:
  url: http://localhost:9074/ds-discover/
```
Not too sure how we should organize it, so let's discuss that at the office.

Reply:
I like the openapi grouping for variables that are only needed for the specifications.